### PR TITLE
Reupdated

### PR DESCRIPTION
**Pull request by apennella**
Re updated everything

### CODE REVIEW

File: Employee.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
+package extraction.sProject;
+import java.util.*;
+public class Employee {
+	
+	private String employeeID;
+	private String employeeName;
+	private String employeeEmail;
+	private double classID;
+	private List<Module> modScores = new ArrayList<>();
```
Comment (kabollinger03) [inline Employee.java]: Does this work? Shouldn't you have the same class name on both sides? Like... If you are going to use an ArrayList, which I recommend then declare the var as such.

File: Employee.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	private String employeeEmail;
+	private double classID;
+	private List<Module> modScores = new ArrayList<>();	
+	
+	public Employee() {
+		super();
+		this.employeeID = "N/A";
+		this.employeeName = "N/A";
+		this.employeeEmail = "N/A";
+		this.classID = 0;
```
Comment (kabollinger03) [inline Employee.java]: Perhaps `-1` is a better dummy value? There may be a class 0 at some point.

File: Module.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+public class Module {
+	
+	private String moduleID;
+	private double moduleScore;
+	
+	public Module() {
+		super();
+		this.moduleID = "N/A";
+		this.moduleScore = 0;
```
Comment (kabollinger03) [inline Module.java]: Same as above, `-1` may be a better default. Except here there should never be a score below 0.

**Review by kabollinger03 — DISMISSED** [review]
No serious issues, just some minor nitpicking

File: Module.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
+package extraction.sProject;
+
+public class Module {
+	
+	
+	private String moduleID;
+	private double moduelsScore;
```
Comment (cgreen9008) [inline Module.java]: It works, but did you mean to name it "moduels" instead of "modules"?

**Review by apj11c — APPROVED** [review]
FINALLY

**Review by cgreen9008 — APPROVED** [review]
Tony made about tree-fiddy changes to this...

**Pull request merged**